docs/usage/quickstart.rst

```diff
@@ -56,105 +116,42 @@ listing all the hosts, aptly called :file:`hosts`:
+   "node-0" with "server".
 
-Upgrading from MetalK8s < 0.2.0
--------------------------------
-MetalK8s 0.2.0 introduced changes to persistent storage provisioning which are
-not backwards-compatible with MetalK8s 0.1. These changes include:
+  .. warning::
 
-- The default LVM VG was renamed from `kubevg` to `vg_metalk8s`.
-- Only *PersistentVolumes* required by MetalK8s services are created by
-  default.
-- Instead of using dictionaries to configure the storage, these are now
-  flattened.
+     Using the remote `root` user to deploy MetalK8s is not supported, see
+     :ref:`ansible-user-root-detection`.
 
-When a MetalK8s 0.1 configuration is detected, the playbook will report an
-error.
+4. Create a :file:`group_vars` subdirectory in the directory you created in
+   step 2 (the one that contains the :file:`hosts` file) and change to it.
 
-Given an old configuration looking like this
+   .. code-block:: shell
 
-.. code-block:: yaml
+    $ mkdir group_vars ; cd group_vars
 
-    metal_k8s_lvm:
-      vgs:
-        kubevg:
-          drives: ['/dev/vdb']
+5. In the :file:`group_vars` subdirectory, create a :file:`kube-node.yml` file.
+   This file declares how to set up hosts in the kube-node group; that is,
+   hosts on which pods shall be scheduled:
 
-the following values must be set in :file:`kube-node.yml` to maintain the
-pre-0.2 behaviour:
+   .. code-block:: yaml
 
-- Disable deployment of 'default' volumes:
-
-  .. code-block:: yaml
-
-      metalk8s_lvm_default_vg: False
-
-- Register the `kubevg` VG to be managed:
-
-  .. code-block:: yaml
-
-      metalk8s_lvm_vgs: ['kubevg']
-
-- Use :file:`/dev/vdb` as a volume for the `kubevg` VG:
-
-  .. code-block:: yaml
-
-      metalk8s_lvm_drives_kubevg: ['/dev/vdb']
-
-  Note how the VG name is appended to the `metalk8s_lvm_drives_` prefix to
-  configure a VG-specific setting.
-
-- Create and register the default MetalK8s 0.1 LVs and *PersistentVolumes*:
-
-  .. code-block:: yaml
+    metalk8s_lvm_drives_vg_metalk8s: ['/dev/vdb']
 
-      metalk8s_lvm_lvs_kubevg:
-        lv01:
-          size: 52G
-        lv02:
-          size: 52G
-        lv03:
-          size: 52G
-        lv04:
-          size: 11G
-        lv05:
-          size: 11G
-        lv06:
-          size: 11G
-        lv07:
-          size: 5G
-        lv08:
-          size: 5G
+   This example assumes every *kube-node* host has a disk available as
+   :file:`/dev/vdb` that can be used to set up Kubernetes *PersistentVolumes*.
+   For more information, see :ref:`storage-architecture`.
 
-Entering the MetalK8s Shell
----------------------------
-To easily install a supported version of Ansible and its dependencies, as well
-as some Kubernetes tools (:program:`kubectl` and :program:`helm`), we provide a
-:program:`make` target which installs these in a local environment. To enter
-this environment, run :command:`make shell` (this takes a couple of seconds on
-first run)::
+Enter the MetalK8s Virtual Environment Shell
```

Comment:
At no point before did you unpack the source zipfile I believe. Should this be added?